templates/blockonomics_checkout.php

```diff
@@ -39,6 +39,11 @@
         <h2><?=__('Could not generate new Bitcoin Cash address', 'blockonomics-bitcoin-payments')?></h2>
         <p><?=__('Note to webmaster: Please follow the instructions <a href="https://help.blockonomics.co/en/support/solutions/articles/33000253348-bch-setup-on-woocommerce" target="_blank">here</a> to configure BCH payments.', 'blockonomics-bitcoin-payments')?></p>
       </div>
+      <!-- Duplicate Address Error -->
+      <div id="address-error-duplicate" ng-show="address_error_duplicate" ng-cloak>
```

Comment:
We are uncesssary creating multiple divs for errrors. I prefer we have a div _customer_facing_error_

app.js can match is string contains gap limit or temporary error and set this div

Reply:
I will update the Duplicate Address Error message. 

It will be better to update to the single customer_facing_error in a separate PR as the translation strings will require passing to js:
https://make.wordpress.org/core/2018/11/09/new-javascript-i18n-support-in-wordpress/

Reply:
Can you finish this so that we can close the PR

Reply:
Refactored to single customer_facing_error in 463c4eb